mlos_bench/mlos_bench/optimizers/base_optimizer.py

```diff
@@ -242,16 +242,24 @@ def bulk_register(self,
         is_not_empty : bool
             True if there is data to register, false otherwise.
         """
-        _LOG.info("Update the optimizer with: %d configs, %d scores, %d status values",
-                  len(configs or []), len(scores or []), len(status or []))
+        _LOG.info(
+            "Update the optimizer with: %d configs, %d scores, %d status values",
+            len(configs or []), len(scores or []), len(status or [])
+        )
```

Comment:
What's the change here?

Reply:
none at all :)

Reply:
@yshady the general rule for submitting PRs (open-source or not) is to separate the functionality changes from cosmetic fixes and make sure each PR is as small as possible (but not smaller)

Reply:
Yeah sorry this is kinda a first for me :) like a kid learning to walk for first time haha

Reply:
let me try to clean this up tomorrow after my demo and explain some of the stuff. I also still need to figure out how to use GitHub correctly